README.rst

```diff
@@ -78,18 +78,18 @@ Dependencies
 
 The minimum required dependencies to run MNE-Python are:
 
-- Python >= 2.7
-- NumPy >= 1.8
-- SciPy >= 0.12
+- Python >= 3.5
+- NumPy >= 1.11.3
+- SciPy >= 0.17.1
 
 For full functionality, some functions require:
 
-- Matplotlib >= 1.3
+- Matplotlib >= 1.5
 - Mayavi >= 4.6
 - PySurfer >= 0.8
-- Scikit-learn >= 0.15 (>= 0.18 recommended)
+- Scikit-learn >= 0.18
 - NiBabel >= 2.1.0
-- Pandas >= 0.13
+- Pandas >= 0.18
```

Comment:
These newer dependencies reflect what I actually had to use to get a working Python 3.5 env on Travis with `conda`. I think they are reasonable numbers at this point anyway

Reply:
+1